lib/src/components/NxTag/NxTag.scss

```diff
@@ -37,22 +38,28 @@
   background-color: var(--nx-selectable-color-light);
```

Comment:
Is the padding-right on the line above this also supposed to have the border subtracted?

Reply:
> Is the padding-right on the line above this also supposed to have the border subtracted?

9c73156f076493c0879e3b303f4d4457150ab4c4